docs/plugin-protocol/tfplugin5.2.proto

```diff
@@ -70,12 +70,20 @@ message RawState {
     map<string, string> flatmap = 2;
 }
 
+enum StringKind {
+    PLAIN = 0;
+    MARKDOWN = 1;
+}
```

Comment:
Is there a reason for introducing Markdown in the context of this particular PR?

Reply:
I answered this below, this was an attempt to minimize protocol version churn, but we could probably have structured the commits / PR to better illustrate the intent.